subsys/bluetooth/mesh/net.c

```diff
@@ -493,7 +501,7 @@ static int loopback(const struct bt_mesh_net_tx *tx, const uint8_t *data,
 
 	err = k_mem_slab_alloc(&loopback_buf_pool, (void **)&buf, K_NO_WAIT);
 	if (err) {
-		BT_WARN("Unable to allocate loopback");
+		LOG_WRN("Unable to allocate loopback");
```

Comment:
@stephanosio The CI is failing because of a compliance failure (`WARNING:EMBEDDED_FUNCTION_NAME`) but this is a false positive. The same is happening with this file: `subsys/bluetooth/host/settings.c:191`. 
Is there a way to fix that?

Reply:
well, a way to "fix" that would be to use a less generic name for the function (e.g. `send_loopback`, `do_loopback`, ...).

The `WARNING:EMBEDDED_FUNCTION_NAME` is often useful and I do not think we should disable it. If there are false positives, then there are false positives. We just need to override the checks when merging.

Reply:
Indeed. A common (no-brainer) prefix for such functions is to simply add the module name as the prefix, i.e. `mesh_loopback` in this case.

Reply:
Well, I remember that the same problem happened in the past, and the discussion at that time was that global functions or any public symbols need to be prefixed, and I think the same applies to this problem, but it is indeed a huge project, but I think it should be  Form a rule to monitor.

Reply:
I changed the functions' names. I welcome better ideas for names.